dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/model/SendContributionFileDataToDrcRequest.java

```diff
@@ -1,16 +1,15 @@
 package uk.gov.justice.laa.crime.dces.integration.model;
 
-import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
-import lombok.NoArgsConstructor;
+import uk.gov.justice.laa.crime.dces.integration.model.generated.contributions.CONTRIBUTIONS;
+
+import java.util.HashMap;
+import java.util.Map;
 
 @Data
 @Builder
-@NoArgsConstructor
-@AllArgsConstructor
 public class SendContributionFileDataToDrcRequest {
```

Comment:
Can we rename this class to something more sensible, while we are making changes here? Something that doesn't start with a verb, e.g. ContributionFileDataRequestForDrc, perhaps?

Reply:
going with `ContributionDataForDrc` (to avoid the phrase `ContributionFile`, which might be confusing, seeing as this is just one record from `concor_contributions` table, not the whole batch from `contributions_files` table).